assets/javascripts/modules/autoComplete.js

```diff
@@ -351,6 +351,17 @@ var suggestionsEvent = function () {
 
 
 /**
+  * If a select elem is being used, load value from there
+**/
+var loadExistingValue = function () {
+    var dataLinked = $('select[id="' + $autoCompleteInputElem.attr("data-select-elem") + '"]');
+    var selectedOption = dataLinked.find('option:selected')[0];
+    if(selectedOption.value > ""){
```

Comment:
funny conditional I presume because value could be a `0`?
IMO `if(selectedOption.value !== '')` a bit easier on the eye